### ISSUE TITLE
Chinese translation is not 100% completed

### ISSUE THREAD

**Issue by mikelyou** [cairoshell/cairoshell#544]
**Describe the bug**
When I use the Chinese version, I found there are many places not translated yet. Like this:
![image](https://user-images.githubusercontent.com/58005137/111451240-2dc3c100-874c-11eb-9c41-aafbd352aff2.png)

And some are not showing correct, like this:
![image](https://user-images.githubusercontent.com/58005137/111451472-6e233f00-874c-11eb-9ba5-d8f5d37d8007.png)
(compared with the English version)
![image](https://user-images.githubusercontent.com/58005137/111451723-b2164400-874c-11eb-9bfb-80ddb4a44bd2.png)

I'm not sure if these are bugs, or translation is not up to date. But without doubt they are affecting the user experience.
If it's the problem of translation and there is no one working on it, maybe I can help. 😃 

**System information:**
 - Windows version: Windows 10 64-bit
 - Cairo version: 0.4.80.38114 - Pre-release
 - In Cairo Settings > Advanced, is Cairo set as shell?:  No

**Comment by dremin** [comment]
These missing translations are due to features added after the Chinese translation was added. Please feel welcome to help contribute to it, if you would like!

**Comment by mikelyou** [comment]
Yes, of course! Please let me know how to contribute. I am happy to help. 😃 
By the way, I am not an expert at programming, it would be better if there are fewer code things for me.
In addition, since it seems I am going to use cario for quite some time in the future, I could help maintain the Chinese translations when big updates come. Hope you will like this.
